### PR TITLE
updated readmes for 1.8 and changed where scripts are downloaded from

### PR DESCRIPTION
**Pull request by bmpotter**
Signed-off-by: Bruce Potter <bp@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,54 +55,62 @@ The SDO owner services are packaged as a single docker container that can be run
...
+For production use of SDO, you need to create 3 key pairs and import them into the owner services container. These key pairs enable you to securely take over ownership of SDO ownership vouchers from SDO-enabled device manufacturers, and to securely configure your booting SDO devices. Use the provided `generate-key-pair.sh` script to easily create the necessary key pairs. (If you are only trying out SDO in a dev/test environment, you can use the built-in sample key pairs and skip this section. You can always add your own key pairs later.)
 
 1. Go to the directory where you want your generated keys to be saved then download `generate-key-pair.sh`.
 
    ```bash
-   curl -sSLO https://raw.githubusercontent.com/open-horizon/SDO-support/master/keys/generate-key-pair.sh
+   curl -sSLO https://github.com/open-horizon/SDO-support/releases/download/v1.8/generate-key-pair.sh
```
Comment (lorenzoking) [inline README.md:77]: just got to replace this file with the updated code including macOS support

File: README.md [hunk 2]
```diff
@@ -94,54 +55,62 @@ The SDO owner services are packaged as a single docker container that can be run
...
    chmod +x generate-key-pair.sh
    ```
    
-2. Run `generate-key-pair.sh` script. You will be prompted to answer a few questions in order to produce corresponding certificates to your private keys. You must be using Ubuntu to run this script.
+2. Run the `generate-key-pair.sh` script. You will be prompted to answer a few questions in order to produce certificates for your private keys. (The prompts can be avoided by setting environment variables. See the script source for details.) You must be using Ubuntu to run this script.
```
Comment (lorenzoking) [inline README.md:81]: you can type `./generate-key-pair.sh -h` to see the environment variables that need to be set as well

File: README.md [hunk 3]
```diff
@@ -155,9 +124,9 @@ The ownership voucher created for the device in the previous step needs to be im
```
Comment (lorenzoking) [inline README.md:124]: ^ I've found the ownership voucher in `~/var/sdo/voucher.json` 

I'm not sure if the `/var` makes a difference or not though

File: README.md [hunk 4]
```diff
@@ -155,9 +124,9 @@ The ownership voucher created for the device in the previous step needs to be im
 
 ### <a name="boot-device"></a>Boot the Device to Have it Configured
 
-When an SDO-enabled device boots, it starts the SDO process which contacts the SDO owner services to configure the device for this Horizon instance. **Back on your VM device**, simulate the booting of the device and watch SDO configure it:
+When an SDO-enabled device boots, it starts the SDO process which first contacts the rendezvous server, which redirects it to the SDO owner services in your Horizon instance. **Back on your VM device**, simulate the booting of the device and watch SDO configure it:
```
Comment (lorenzoking) [inline README.md:127]: I think breaking the first sentence into two helps with understanding the flow.

When an SDO-enabled device boots, it starts the SDO process which first contacts the rendezvous server. This rendezvous server then redirects it to the SDO owner services in your Horizon instance. **Back on your VM device**, simulate the booting of the device, and watch SDO configure it:

File: README.md (hunk 1 above)
Comment (bmpotter) [reply README.md:77]: yes

File: README.md (hunk 3 above)
Comment (bmpotter) [reply README.md:124]: Hmm, i find it in `/var/sdo/voucher.json` so i'm changing the doc to that. Thx for catching this.

File: README.md (hunk 4 above)
Comment (bmpotter) [reply README.md:127]: good suggestion

**Review by lorenzoking — APPROVED** [review]

**Pull request merged**